src/main/java/eventprocessing/models/InitialBucket.java

```diff
@@ -10,7 +10,7 @@ public InitialBucket() {
     }
 
     public long getEarliestTimestamp() {
-        Optional<SqsResponse> earliestResponse = getSqsResponse().stream()
+        Optional<SqsResponse> earliestResponse = getSqsResponses().stream()
```

Comment:
Can `InitialBucket` and `BucketManager` now be deleted? (Along with the extra constructor in `Bucket`?)